requirements-dev.txt

```diff
@@ -3,5 +3,7 @@ pytest>=7.0.0
 pytest-asyncio>=0.18.0
 pytest-cov>=3.0.0
 pytest-mock>=3.6.0
+mypy>=1.0.0
```

Comment:
I'd highly suggest pinning these to specific versions and then using Dependabot to update them automatically. That way you'll know what change is breaking something in your CI, rather than ending up with several breakages in master when you come back after a few months.

You can probably just use these if you want to set it up:
https://github.com/Dreamsorcerer/gnucash-gocardless-import/blob/master/requirements.txt
https://github.com/Dreamsorcerer/gnucash-gocardless-import/blob/master/.github/dependabot.yml
https://github.com/Dreamsorcerer/gnucash-gocardless-import/blob/master/.github/workflows/auto-merge.yml